Formula/numpy.rb

```diff
@@ -37,7 +39,7 @@ def install
     ENV.prepend_create_path "PYTHONPATH", Formula["cython"].opt_libexec/"lib/python#{xy}/site-packages"
 
     system Formula["python@3.9"].opt_bin/"python3", *Language::Python.setup_install_args(prefix),
```

Comment:
```suggestion
    system Formula["python@3.9"].opt_bin/"python3", "build", "--fcompiler=gfortran", "--parallel=#{ENV.make_jobs}"
    system Formula["python@3.9"].opt_bin/"python3", *Language::Python.setup_install_args(prefix)
```
I think we can align this with the `scipy` formula.